qa/suites/rgw/verify/validater/valgrind.yaml

```diff
@@ -13,8 +13,9 @@ overrides:
       mon:
         mon osd crush smoke test: false
     valgrind:
-      mon: [--tool=memcheck, --leak-check=full, --show-reachable=yes]
       osd: [--tool=memcheck]
       mds: [--tool=memcheck]
+      # mon commented out due to: http://tracker.ceph.com/issues/38827
```

Comment:
Since a formatting style for commenting out has been established immediately below, you might as well use the same one.